client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import toString from 'lodash/toString';
+
+const reactifyNewlines = message =>
+	message
+		.split( '\n' )
+		.map( ( snip, index ) => [ <span key={ `s${ index }` }>{ snip }</span>, <br key={ `br${ index }` } /> ] )
+		.reduce( ( final, next ) => final.concat( next ), [] )
+		.slice( 0, -1 );
+
+export const SeoSiteSearchPreview = React.createClass( {
```

Comment:
it's a habit of mine. I can take it out, but the way I usually end up writing tests is to import the component by name `import { SeoSiteSearchPreview } from …` because that's the non-connected component whereas the default _might_ be connected to the Redux tree.